src/view/frontend/web/js/cart-tracking.js

```diff
@@ -0,0 +1,45 @@
+define(['factfinder', 'jquery', 'Magento_Customer/js/customer-data'], function (factfinder, jQuery, customerData) {
```

Comment:
Why dont we rewrite this as a mixin of the add-to-cart script? Like this, our code will be loaded automatically every time the other one is loaded

Reply:
Alternatively, we could do a mixin of the cart object and subscribe there, then we have the real data every time.

Reply:
I found` catalog-add-to-cart.js` cumbersome to hook into via mixin. The method i should override would be `ajaxSubmit` which would require copy all the code from parent if i want to fire tracking at correct moment (ajax success handler).
There would be lot easier if successHandler will be a method of the widget.
Perhaps you have some other solution in mind - we can discuss it